src/login/AccountActivationMessage.jsx

```diff
@@ -23,12 +24,12 @@ const AccountActivationMessage = (props) => {
 
   switch (messageType) {
     case ACCOUNT_ACTIVATION_MESSAGE.SUCCESS: {
-      heading = intl.formatMessage(messages['account.activation.success.message.title']);
-      activationMessage = intl.formatMessage(messages['account.activation.success.message']);
+      heading = intl.formatMessage(messages[`account.${isVerificationMessage ? 'verification' : 'activation'}.success.message.title`]);
```

Comment:
Can we make it a variable and use that instead in all three places, it will make the code cleaner.
```
activationOrVerification =  getConfig().MARKETING_EMAILS_OPT_IN ? 'verification' : 'activation';
intl.formatMessage(messages[`account.${activationOrVerification}.success.message.title`]);
```

Reply:
Yes makes sense. I will update it